monai/handlers/segmentation_saver.py

```diff
@@ -101,13 +101,12 @@ def __init__(
                 output_dir: /output,
                 data_root_dir: /foo/bar,
                 output will be: /output/test1/image/image_seg.nii.gz
-            batch_transform: a callable that is used to transform the
-                ignite.engine.batch into expected format to extract the meta_data dictionary.
-                it can be used to extract the input image meta data: filename, affine, original_shape, etc.
-            output_transform: a callable that is used to transform the
-                ignite.engine.output into the form expected image data.
-                The first dimension of this transform's output will be treated as the
-                batch dimension. Each item in the batch will be saved individually.
+            batch_transform: a callable that is used to extract the `meta_data` dictionary of the input images
+                from `ignite.engine.batch`. then extract necessary information from the meta data: filename,
+                affine, original_shape, etc.
+            output_transform: a callable that is used to extract the model prediction data from
```

Comment:
my suggestion is:
output_transform: a callable that is used to transform the segmentation predictions achieved from `ignite.engine.state.output`. The first dimension of its output will be treated as the batch dimension and each item in the batch will be saved individually.

Reply:
updated in the latest commit.
Thanks.